kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/ForumStatistics.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.testing.forum.statistics;
+
+public class ForumStatistics {
+    private Statistics Statistics;
+
+    public ForumStatistics(Statistics statistics) {
+        this.Statistics = statistics;
+    }
+
+    String calculateAdvStatistics(Statistics statistics) {
+        int usersQuantity = Statistics.usersNames().size();
+        int postsCount = Statistics.postsCount();
+        int commentsCount = Statistics.commentsCount();
+        int postsPerUser = Statistics.postsCount() / usersQuantity;
```

Comment:
to powinien byc double

Reply:
zwroc uwage na konwersje typow int/int -> int -> double (tracisz wartosci po przecinku). Wykorzystaj np. rzutowanie `zmiennaA/(double)zmiennB`